.github/workflows/pull-request-ci.yaml

```diff
@@ -306,6 +306,27 @@ jobs:
       - run: make acceptance-tests
         timeout-minutes: 180
 
+      # Rotate CLoud Controller Database
+      - name: Rotate CCDB
+        id: ccdb-rotate
+        run: |
+          source ${GITHUB_WORKSPACE}/catapult/build${CLUSTER_NAME}/.envrc
+          testing/ccdb_key_rotation/rotate-ccdb-keys-test.sh
+
+          echo "::group::Wait for KubeCF to be ready again"
+          make kubecf-wait
+          echo "::endgroup::"
+        env:
+          CHART: ${{ github.workspace }}/kubecf-bundle/kubecf_release.tgz
+        continue-on-error: ${{ !matrix.ccdb-rotate }}
```

Comment:
From what I can read, `continue-on-error` marks the step as green, even if it fails, which is dangerous.
I wonder if this line is needed if `fail-fast: false` is set.

Perhaps we want to set `fail-fast: true` and:
```
run: foo
if: experimental == true
```

But I may be misunderstanding things.
(same for the other instances of `continue-on-error` and the re-do of the experimental jobs in the PR).

Reply:
I see, thanks for the detailed explanation :).
It's just that `continue-on-error` always marks the step as green on UI, so we need to aware of that.